convokit/tests/general/metadata_operations/test_corpus_index_meta.py

```diff
@@ -132,7 +146,9 @@ def setUp(self) -> None:
             Corpus(
                 utterances=[
                     Utterance(id="0", text="hello world", speaker=Speaker(id="alice")),
-                    Utterance(id="1", text="my name is bob", speaker=Speaker(id="bob")),
+                    Utterance(
+                        id="1", text="my name is bob", reply_to="0", speaker=Speaker(id="bob")
+                    ),
```

Comment:
Why was this change only made to `TestWithDB`? I assume you meant to make the same change in `TestWithMem` as well; as it currently stands the result is that mem mode and DB mode are being tested on corpora with slightly differing structures, which was presumably not the intent.

Reply:
Relatedly, several shared test functions such as `key_insertion_deletion` try to look up a Conversation with ID `convo_id_0` in `self.corpus`, despite the fact that no such conversation exists in `self.corpus` as defined in either of the `setUp` methods

Reply:
Great catch -- introduced some bugs when refactoring these tests as part of #177. Should be resolved now.